config/locales/en/lookup.yml

```diff
@@ -10,8 +10,10 @@ en:
         input_error: Enter a valid postcode
         error_description: Enter a valid postcode
     lookup:
-      input_label: Enter your postcode
-      hint_text: For example LS1 1UR
+      input_label: <h2 class="govuk-heading-s"> Enter a full postcode </h2>
+      hint_text: |
+        <p class="govuk-hint"> We use this to give you more accurate information about the area.</p>
```

Comment:
we should just have the text here and let the view handle the classes etc

Reply:
Was this resolved via a conversation? Why is the `govuk-hint` class needed here?

Reply:
Ok, I played around with this locally, and figured out why the `govuk-hint` class is needed. Can you add a note to the commit message about this @GDSNewt?

Reply:
> Was this resolved via a conversation? Why is the `govuk-hint` class needed here?

Ahhh I prematurely resolved the conversation when I removed the html from the input_label yaml. As for the gov-hint class in the hint text, yeah it is needed to have the correct spacing between the two lines of the hint. We adopt a similar approach for the body text and inset text on this page so I thought this was ok to do so in this instance. I'm not really sure how you would do it via the view, but happy to look into it and change it for this, the body text and the inset text if that is the approach we are taking.

Reply:
I wonder if someone like @DilwoarH or @andysellick knows of another way to add multiline hint text?

Reply:
I'll look into it